src/pages/SharePage.js

```diff
@@ -0,0 +1,177 @@
+                type: PropTypes.string.isRequired,
+
+                // File URI
+                uri: PropTypes.string.isRequired,
+            }),
+        ]).isRequired,
+    }),
+};
+
+const defaultProps = {
+    currentlyViewedReportID: '',
+    sharedItem: null,
+};
+
+class SharePage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isCreateMenuActive: false,
+        };
+
+        this.onCloseButtonClick = this.onCloseButtonClick.bind(this);
+        this.onLinkClick = this.onLinkClick.bind(this);
+        this.onCreateMenuItemSelected = this.onCreateMenuItemSelected.bind(this);
+        this.toggleCreateMenu = this.toggleCreateMenu.bind(this);
+        this.addSharedItemToReport = this.addSharedItemToReport.bind(this);
+    }
+
+    /**
+     * Clears the shared item and closes Share Page.
+     */
+    onCloseButtonClick() {
+        clearSharedItem();
+        redirect(this.props.currentlyViewedReportID !== ''
+            ? ROUTES.getReportRoute(this.props.currentlyViewedReportID)
+            : ROUTES.HOME);
+    }
+
+    /**
+     * Hides navigation menu on redirect to report page.
+     */
+    onLinkClick() {
+        hideSidebar();
+    }
+
+    /**
+     * Method called when a Create Menu item is selected.
+     */
+    onCreateMenuItemSelected() {
+        this.toggleCreateMenu();
+        ChatSwitcher.show();
+    }
+
+    /**
+     * Posts shared item to selected report
+     *
+     * @param {Number} reportID report id
+     */
+    addSharedItemToReport(reportID) {
+        if (!this.props.sharedItem) {
+            return;
+        }
+
+        switch (this.props.sharedItem.type) {
+            case ShareManager.TYPE.TEXT:
+                addAction(reportID, this.props.sharedItem.data);
+                break;
+            case ShareManager.TYPE.FILE:
+                addAction(reportID, '', this.props.sharedItem.data);
+                break;
+            default:
+                break;
+        }
+
+        clearSharedItem();
+    }
+
+    /**
+     * Method called when we click the floating action button
+     *
+     * Method called either when:
+     * Pressing the floating action button to open the CreateMenu modal
+     * Selecting an item on CreateMenu or closing it by clicking outside of the modal component
+     */
+    toggleCreateMenu() {
+        this.setState(state => ({
+            isCreateMenuActive: !state.isCreateMenuActive,
+        }));
+    }
+
+    render() {
+        return (
+            <SafeAreaProvider>
+                <CustomStatusBar />
+                <SafeAreaInsetsContext.Consumer style={[styles.flex1]}>
+                    {insets => (
+                        <>
+                            <View style={[styles.flex1, styles.sidebar]}>
+                                <SidebarLinks
+                                    title="Send to..."
+                                    showAvatar={false}
```

Comment:
You can remove this line as per the other comment about showAvatar being false by default.